### DISCUSSION TITLE
ArcadeDB Wikipedia Page

### DISCUSSION THREAD

**Discussion by gramian** [ArcadeData/arcadedb#540] [General]
Dear ArcadeDB Community,

I was wondering, if ArcadeDB should have a Wikipedia page? Wikipedia references ArcadeDB already multiple times (see: https://en.wikipedia.org/w/index.php?search=%22ArcadeDB%22&title=Special%3ASearch&profile=advanced&fulltext=1&ns0=1 ), and to be listed in the tables on pages like [NoSQL](https://en.wikipedia.org/wiki/NoSQL) requires a Wikipedia page.

Cheers

BTW: a Wikidata item for ArcadeDB exists already: https://www.wikidata.org/wiki/Q113997665

**Comment by lvca** [comment]
It would be awesome having an official page on Wikipedia, especially from our users. Any help is more than welcome.

**Reply by gramian** [reply]
I started a draft page for ArcadeDB on Wikipedia: https://en.wikipedia.org/wiki/Draft:ArcadeDB

**Reply by lvca** [reply]
That's great @gramian. I hope it could be accepted, sometimes product pages are banned by the reviewer without comments and it's hard to get passed that.
